ramls/template-engine.raml

```diff
@@ -2,7 +2,7 @@
 
 title: Template engine
 baseUri: http://api.example.com/{version}
-version: v2.1
+version: v2.2
```

Comment:
Please change interface version in descriptor as well

Reply:
fixed